resources/themes/STARTERTHEME/Gruntfile.js

```diff
@@ -13,10 +13,13 @@ module.exports = function (grunt) {
         files: [{
           expand: true,
           cwd: 'sass',
-          src: ['*.scss', '**/*.scss'],
+          src: '**/*.scss',
```

Comment:
We should add a static sass root folder.
No need to scan through all theme folders for sass files. (node_modules would interfere for example).